Assets/Project/Modules/PlayerAnchor/Scripts/Anchor/VFXAnchorView.cs

```diff
@@ -116,7 +117,12 @@ public async UniTaskVoid PlayThrownAnimation(float duration)
         {
             StopCarry();
             
-            _particleFactory.Create(_throwHeadParticleType, Vector3.zero, Quaternion.identity, _vfxParent);
+            Transform head = _particleFactory.Create(_throwHeadParticleType, Vector3.zero, Quaternion.identity, _vfxParent);
+            
+            //Used for different way to move the effect pending of more feedback in order to make a decision
+            //head.transform.parent = _anchorThrowHeadFollower.transform;
+            //_anchorThrowHeadFollower.gameObject.SetActive(true);
```

Comment:
Call `_anchorThrowHeadFollower.StartFollowing()` instead.

Instead of commenting, add an IF with an exposed bool variable.

And then always call `_anchorThrowHeadFollower.StopFollowing()` at the end of the effect

Reply:
After adding the comment about the "VFXAnchorViewConfig" script, the bool I mention in the comment above should also be there in that script.

Reply:
Done 👌